### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by dwillmer**

### CODE REVIEW

File: src/plugin.ts [hunk 1]
```diff
@@ -276,7 +276,7 @@ function loadPackage(name: string): Promise<void> {
  */
 function addPackage(name: string, config: any) {
   if (config.hasOwnProperty('phosphor-plugin')) {
-    var pconfig = config['phosphor-plugin'] as IPluginJSON;
+    var pconfig = config["phosphor-plugin"] as IPluginJSON;
```
Comment (sccolbert) [inline src/plugin.ts]: why the quote change?

File: .gitignore [hunk 2]
```diff
@@ -2,7 +2,6 @@
 *.user
 .DS_Store
 docs/api
-lib
```
Comment (blink1073) [inline .gitignore]: I'd leave this here for when we actually publish.

File: src/plugin.ts (hunk 1 above)
Comment (dwillmer) [inline src/plugin.ts]: Will fix, was actually a change from 'phosphide' to "phosphor-plugin", hidden by the re-merge.

**Comment by blink1073** [comment]
:+1:

**Pull request merged**